libr/cons/input.c

```diff
@@ -473,6 +473,10 @@ R_API int r_cons_yesno(int def, const char *fmt, ...) {
 		key = 'y';
 	r_cons_set_raw (0);
 	if (key=='\n' || key=='\r')
+		/* I don't know why but with this eprintf
+		 * the Y/N functionality works properly at MIPS
+		 */
+		eprintf("");
```

Comment:
also this is a bug because not using {} introduced a regression because key = def will always run. are you a pythonist? :P